keyboards/planck/keymaps/rjhilgefort/keymap.c

```diff
@@ -0,0 +1,167 @@
+* |-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+* |       | Trak ← | Trak → | Scrn - | Scrn + |        |        |   1    |   2    |   3    |        |        |
+* |-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+* |       |        |        |        |        |                 |        |   0    |   .    |        |        |
+* `----------------------------------------------------------------------------------------------------------'
+*/
+[_RAISE] = {
+  {_______, _______, _______, _______, _______, _______, _______, KC_7,    KC_8,    KC_9,    _______, _______},
+  {_______, KC_MUTE, KC_VOLD, KC_VOLU, KC_MPLY, _______, _______, KC_4,    KC_5,    KC_6,    _______, _______},
+  {_______, KC_MRWD, KC_MFFD, KC_SLCK, KC_PAUS, _______, _______, KC_1,    KC_2,    KC_3,    _______, _______},
+  {_______, _______, _______, _______, _______, _______, _______, KC_0,    KC_0,    KC_DOT,  _______, _______}
+},
+
+/* Adjust (Lower + Raise)
+ * ,----------------------------------------------------------------------------------------------------------.
+ * |       | Reset  |        |        |        |        |        |        |        |        |        | Del    |
+ * |-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |       |        | MuMod  | AudOn  | AudOff |        |        | Qwerty |        |        |        |        |
+ * |-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |       | Voice- | Voice+ | MusOn  | Musoff | MIDIon | MIDIof | TermOn | TermOff|        |        |        |
+ * |-------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |       |        |        |        |        |                 |        |        |        |        |        |
+ * `----------------------------------------------------------------------------------------------------------'
+ */
+[_ADJUST] = {
+  {_______, RESET,   _______, _______, _______, _______, _______, _______, _______,  _______, _______, KC_DEL },
+  {_______, _______, MU_MOD,  AU_ON,   AU_OFF,  _______, _______, QWERTY,  _______,  _______, _______, _______},
+  {_______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  TERM_ON, TERM_OFF, _______, _______, _______},
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______}
+}
+
+};
+
+#ifdef AUDIO_ENABLE
+  float plover_song[][2]     = SONG(PLOVER_SOUND);
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
```

Comment:
If you're not using any other layouts, then this can be changed/removed  completely, actually.